bin/lrs-test.js

```diff
@@ -6,10 +6,12 @@
 (function (process, require, program, exit, packageJson, Q, Joi, fs, path, Mocha) {
     'use strict';
 
+    var DIRECTORY = 'v1_0_2';
```

Comment:
I see we have this value in environment variables also. Should we be pulling it from there here?